fbw-a380x/src/systems/instruments/src/MFD/pages/common/Footer.tsx

```diff
@@ -28,17 +29,33 @@ export class Footer extends DisplayComponent<AbstractMfdPageProps> {
         this.props.fmcService.master.fmsErrors.sub((_, __, ___, arr) => {
           const ind = arr.findIndex((el) => !el.cleared);
 
-          if (ind > -1 && this.messageRef.getOrDefault()) {
+          if (ind > -1 && this.line1Ref.getOrDefault() && this.line2Ref.getOrDefault()) {
             this.messageToBeCleared.set(true);
-            this.messageRef.instance.textContent = arr[ind].messageText;
+
+            let carriageReturn = arr[ind].messageText.search('\n');
```

Comment:
FYI this is a linefeed. Carriage return is `\r`.

Reply:
Of course 😆 gonna fix it